src/Core/Core.csproj

```diff
@@ -34,11 +34,13 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Quantum.Canon" Version="0.5.1904.1302" />
```

Comment:
Just curious, what are you using from this package? I don't think `Microsoft.Quantum.Canon` is what you want, as that one has been deprecated for quite some time.

Reply:
If you're looking for `typeof(Microsoft.Quantum.Canon.ApplyToEach)`, you'll want Microsoft.Quantum.Standard here instead. That said, the libraries repo depends on IQ# such that that then creates a circular dependency between repositories.

Reply:
Yea, @cgranade is right, it was to resolve `Microsoft.Quantum.Canon.ApplyToEach`. I did import `Microsoft.Quantum.Standard` in the first commit, but that broke some tests. Any ideas on why that broke it vs using `Microsoft.Quantum.Canon` would be appreciated!

I guess moving it to `qsharp-runtime` would help solve this. cc-ing @anpaz-msft regarding thoughts on this.

Reply:
@theRoughCode: The Q# libraries are my area, so I'm happy to help with and to discuss. The short version is that we have sometimes moved operations from the Q# standard libraries into the Q# core library to make them available to the simulation runtime and to IQ# (e.g.: `MResetZ`). This doesn't have any user-facing effect, as both the core and standard libraries are made available to Q# applications, but can help with dependency ordering. From that perspective, moving ApplyToEach* to the core so that they can be used here is a completely reasonable suggestion.

The flip side is that it's always a bit of an awkward cross-repo dance to move something from standard to core, so I'd want to be quite sure that's the best solution before proceeding.

Reply:
@cgranade Yea, that makes sense. I was talking to Andres on a sync earlier and he suggested I restructure the core logic surrounding that parsing of each operation into the metadata `Operation` (i.e. `ParseOperation` function) into qsharp-runtime. I'll be looping you in as I make progress on that front. This PR might be reworked a bit till then, so I'll make this a draft PR for now to signify that.